authors/apps/authentication/backends.py

```diff
@@ -41,14 +41,15 @@ def _authenticate_credentials(self, request, token):
         successful, return the user and token. If not, throw an error.
```

Comment:
Great work Edgar. However, there are two commits on your PR.

Reply:
Thank you. I have squashed the two commits into one.